server/src/main/scala/controllers/Application.scala

```diff
@@ -173,6 +174,20 @@ trait SystemActors {
 
   VoyageManifestsProvider(bucket, airportConfig.portCode, crunchInputs.manifests, voyageManifestsActor).start()
 
+
+  def createSplitsPredictionStage(predictSplits: Boolean, rawSplitsUrl: String): SplitsPredictorBase = if (predictSplits)
+    new SplitsPredictorStage(airportConfig.portCode, createSparkSession(), rawSplitsUrl)
+  else
+    new DummySplitsPredictor()
+
+  def createSparkSession(): SparkSession = {
+    SparkSession
+      .builder
+      .appName("Simple Application")
```

Comment:
not sure if this name matters, but possibly not very descriptive.

Reply:
Good spot. I've given it a more appropriate name now :)